### DISCUSSION TITLE
How useful would a PDF editor be?

### DISCUSSION THREAD

**Discussion by SethBurkart123** [SethBurkart123/EvenBetterSEQTA#126] [Ideas]
Something like working with this example:

https://tldraw.dev/examples/use-cases/pdf-editor

Where we allow people to edit pdf's if they want?? So that if it's a pdf file it can open a preview and let them edit it. Because personally I really like using those controls for it... 

I wouldn't want to make it too invasive though.

**Comment by MEGA-Dawg68** [comment]
It could be cool but how big would the storage requirements be?

**Reply by SethBurkart123** [reply]
It'd be probably a bit less than 70kb, so not much of an increase for such a crazy feature...

**Reply by MEGA-Dawg68** [reply]
Well then it isn't that bad. I all for it

**Comment by ar-cyber** [comment]
I'm all into this. My school makes us use Edge's crappy PDF viewer and one built into SEQTA would be nice.

**Reply by MEGA-Dawg68** [reply]
> I'm all into this. My school makes us use Edge's crappy PDF viewer and one built into SEQTA would be nice.

my school tries so hard to make us use edge. But they cannot make me use it, I will fight until the end of time lol
